### DISCUSSION TITLE
Disablew `pane::CloseActiveItem` when tab is pinned

### DISCUSSION THREAD

**Discussion by mil-ad** [zed-industries/zed#22287] [Feature Requests]
I have `"ctrl-w": "pane::CloseActiveItem"` in my bindings but I don't want it to apply to pinned tabs (the same way that you can't just close a pinned tab with mouse before unpinning it). Is this possible?

**Comment by JosephTLyons** [comment]
Thanks for reaching out. We want to better support our community, and the best way for us to do that is through our official Discord. The team is active there and happy to help.

We’re closing the “Help and General Q&A” category on GitHub Discussions and directing all support questions to Discord going forward.

https://zed.dev/support
